### PR TITLE
added carousel

### PR DESCRIPTION
**Pull request by anbarisker**

### CODE REVIEW

**Comment by teochengyong** [comment]
@anbarisker may need to see why both carousel is moving
image

**Comment by anbarisker** [comment]
@teochengyong i have fixed the on the issue of both carousel is moving when clicking next. Need to add swiper container and then add additional class for the pagination and next/prev button

**Review by teochengyong — APPROVED** [review]
LGTM!

File: apps/src/sass/pages/_home.scss [hunk 1]
```diff
@@ -34,12 +50,42 @@
   }
   &-stack-2022-gradient {
     background: linear-gradient(253.82deg, #ff7920 11.24%, #b41e8e 88.76%);
-    padding-top: 0px;
-    padding-bottom: 96px;
+    // Uncomment if using back static banner and not carousel
+    // padding-top: 0px;
+    // padding-bottom: 96px;
```
[the remark this answers is not in the document]
Comment (anbarisker) [reply apps/src/sass/pages/_home.scss]: okay sure, will remove this.

**Pull request merged**